src/main/java/de/jtheb/fh/lagerverwaltung/entities/Compartment.java

```diff
@@ -41,12 +41,14 @@ public int getItemsVolume() {
     }
 
     /**
-     * This Method tests for itemFitsVolume and articleNrFits
+     * This Method tests for itemFitsVolume and itemFits
+     * itemFits tests whether there is an item in Compartment, which fits with articleNr or if the compartment is empty (==null)
+     * itemFitsVolume tests the obvious
```

Comment:
Der Punkt ist mir extrem wichtig. "Obvious" kann falsch verstanden werden, dass jemand zu dumm wäre den Code zu verstehen. Wenn du irgendwo einen Bug hast, und in der Dokumentation steht das wäre doch offensichtlich, ist das beim fixen verdammt nervig. "Er schreibt hin es wäre obvious, obwohl sein Code nicht funktioniert".